packages/example-app-base/src/components/ComponentPage/ComponentPage.scss

```diff
@@ -171,9 +178,26 @@ $dontColor: #A61E22;
     &:first-child {
       @include ms-margin-right(120px);
     }
+
+    .ComponentPage-doSectionHeader {
+      margin: 16px 0 0 0;
+
+      h3 {
+        margin: 0;
+      }
+    }
+
+    .ComponentPage-doSectionHr {
```

Comment:
Can you expand 'Hr' into 'Header' or what ever it's for?

Reply:
It's for the `<hr>` tag that used to be the `:after` on the header since it now has to be inside a container for the edit button.

Reply:
Ah, makes sense now! Maybe another word then? Like 'line' or something? In case it ends up not being an hr tag someday.

Reply:
Good suggestion